oval_graph/client.py

```diff
@@ -182,83 +182,116 @@ def get_src(self, src):
         FIXTURE_DIR = os.path.join(_dir, src)
         return str(FIXTURE_DIR)
 
-    def _prepare_all_in_one_data(self, rules, dict_oval_trees, out, date=None):
+    def _prepare_data(self, rules, dict_oval_trees, out, date):
         for rule in rules['rules']:
             try:
-                self._put_to_dict_oval_trees(dict_oval_trees, rule, date)
+                self._put_to_dict_oval_trees(dict_oval_trees, rule)
+                if not self.all_in_one:
+                    src = self._get_src_for_one_graph(rule, date)
+                    self.save_html_and_open_html(
+                        dict_oval_trees, src, dict(rules=[rule]), out)
+                    dict_oval_trees = {}
             except NotChecked as error:
                 self.print_red_text(error)
-        src = self.get_save_src('rules')
-        self.save_html_with_all_rules_in_one(
-            dict_oval_trees, src, rules, out)
+        if self.all_in_one:
+            src = self.get_save_src('rules' + date)
+            self.save_html_and_open_html(
+                dict_oval_trees, src, rules, out)
         return out
 
-    def _prepare_data_by_one(self, rules, dict_oval_trees, out, date=None):
-        for rule in rules['rules']:
-            try:
-                oval_tree_dict = self.create_dict_of_rule(rule)
-                src = self._get_src_for_one_graph(rule, date)
-                self.save_html_and_open_html(
-                    oval_tree_dict, src, rule, out)
-            except NotChecked as error:
-                self.print_red_text(error)
+    def prepare_data(self, rules):
+        out = []
+        oval_tree_dict = dict()
+        date = str(datetime.now().strftime("-%d_%m_%Y-%H_%M_%S"))
```

Comment:
and while we are at it, this format of time is used project-wide, so I would also suggest to make it a constant. Not necessarily in this PR though.

Reply:
I'll fix it in #131